mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -4376,7 +4376,7 @@ pub(crate) fn tendermint_tx_internal_id(bytes: &[u8], token_id: Option<BytesJson
 }
 
 #[cfg(test)]
-pub mod tendermint_falsecoin_tests {
+pub mod tests {
```

Comment:
This seems like was renamed unintentionally in https://github.com/KomodoPlatform/komodo-defi-framework/pull/2223.